### PR TITLE
TensorFlow 2.0 MNIST example, without IT

### PR DESCRIPTION
**Pull request by liuzhe-lz**
Some example trials depend on `tensorflow.examples`, which is not available in `tf.compat.v1`.
So let integration test environment stick to TF 1.x for now.

### CODE REVIEW

File: examples/trials/mnist-tfv2/mnist.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        y_train,
+        batch_size=params['batch_size'],
+        epochs=10,
+        verbose=0,
+        callbacks=[ReportIntermediates()],
+        validation_data=(x_test, y_test)
+    )
+    _logger.info('Training completed')
+
+    result = model.evaluate(x_test, y_test, verbose=0)
```
Comment (ultmaster) [inline examples/trials/mnist-tfv2/mnist.py]: write `loss, acc = model.evaluate...` here?

File: examples/trials/mnist-tfv2/config_pai.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    #choice: maximize, minimize
+    optimize_mode: maximize
+trial:
+  command: python3 mnist.py
+  codeDir: .
+  gpuNum: 0
+  cpuNum: 1
+  memoryMB: 8196
+  #The docker image to run nni job on pai
+  image: msranni/nni:latest
```
Comment (ultmaster) [inline examples/trials/mnist-tfv2/config_pai.yml]: Tensorflow 1.x is installed in msranni/nni right?

Comment (liuzhe-lz) [reply examples/trials/mnist-tfv2/config_pai.yml]: I think so. Because nobody has upgraded it.

Comment (ultmaster) [reply examples/trials/mnist-tfv2/config_pai.yml]: So is it appropriate for us to put an example here that doesn't even work?

Comment (liuzhe-lz) [reply examples/trials/mnist-tfv2/config_pai.yml]: I removed docker-based config files. Now this example does work, but only supports limited training services.

Comment (liuzhe-lz) [reply examples/trials/mnist-tfv2/config_pai.yml]: Because distributed platforms are not supported, I moved this example down in docs...

**Pull request merged**